fundamental-concepts/data-structures-stdlib/1_records.md

```diff
@@ -0,0 +1,87 @@
+One of the most basic data types in DAML is the record. Here's an example:
+
+<pre>
+data Address = Address {
+  street: Text,
+  city: Text,
+  country: Text
+}
+</pre>
+
+You define a record by listing its field names together with their types. Records can be nested:
+
+<pre>
+data Person = Person {
+  name: Text,
+  address: Address,
+  age: Int
+}
+</pre>
+
+You can create a new `Person` record by assigning each field with a value:
+
+<pre>
+testPerson: Person
+testPerson = Person {name = "Alice", address = Address {street = "RabbitStreet", city = "QueenOfHearts", country = "Wonderland"}, age = "7"}
+</pre>
+
+When you write a function and you need to **deconstruct** a record you can **pattern match**:
+
+<pre>
+prettyAddress: Address -> Text
+prettyAddress (Address aStreet aCity aCompany) = aStreet <> " - " <> aCity <> " - " <> aCompany
```

Comment:
You need to say what `<>` does somewhere here.

Reply:
Agreed, added a sentence.